.github/workflows/go.yml

```diff
@@ -41,6 +41,11 @@ jobs:
               run: go test ./gh
             - name: Test (git)
               run: go test ./git
+            - name: Setup Miniconda
```

Comment:
Seeing that this isn't a test job, I would suggest moving it to the test job bundle.

Reply:
Oh wait, I see the issue. Please pull from master to update your go.yml. I separated the test jobs from the build jobs.

Reply:
Yeah I'll change this when I pull them into master.